index.html

```diff
@@ -1,126 +1,81 @@
-        color : white;
-      }
-      #description {
-        width : 80%;
-        margin : auto;
-        margin-top : 2%;
-        border-right : 1px solid #006400;
-        border-left : 1px solid #006400;
-        padding : 0 5%;
-      }
-      li a {
-        text-decoration : none;
-        color : inherit;
-      }
-      #challenges ul {
-        list-style-type: none;
-        padding : 0;
-        /*margin-top : -10px;*/
-        /*background-color: blue;*/
-      }
-      #challenges ul li {
-        width : 49.8%;
-        display: inline-block;
-        /*background-color: red;*/
-        margin : 0;
-        padding : 0;
-      }
-      #challenges img {
-        width : 100%;
-      }
-      #attendies {
-        background-color: #006400;
-        color : white;
-        text-align: center;
-        padding : 2%;
-      }
-      #attendies a {
-        color : inherit;
-      }
-    </style>
-    <div class="banner">
-      <a target="_blank" href="https://www.facebook.com/events/1798201603730103"><img src="./img/banner.png"></a>
-    </div>
-    <div class="navbar">
-      <ul>
-        <li><a href="#description">Description</a></li>
-        <li><a href="#challenges">Challenges</a></li>
-        <li><a href="#attendies">Attendies</a></li>
-      </ul>
-    </div>
-    <div class="info">
-      <div id="description">
-         <p>
-In this meetup, you'll be forming teams and competing against others to solve coding challenges in JavaScript. We'll use websites like Hacker Rank, Code Fights, Code Wars, Coder Byte and FreeCodeCamp's Algorithm Challenges.
-This will be done in the pair programming format. In case you're not familiar, pair programming is a coding technique in which two people work together on a single system. One, the driver, writes code while the other, the navigator, reviews it. The two programmers switch roles frequently.
-<br><br>
-The goal is to create an comfortable collaborative environment where everyone can learn by competing against each other.
-<br><br>
-Location : <a href="http://goo.gl/WkFCDO" target="_blank">Fission Labs</a>
-<br><br>
-Note :
-<ul>
- <li>This meetup is free of cost.</li>
- <li>Laptops are mandatory, charging ports will be available.</li>
- <li>You can walk in and out at any time during the meetup</li>
-</ul>
-<br>
-FAQ : <br>
-Q. What is FreeCodeCamp? <br>
-A. FreeCodeCamp is a free learning to code website that teaches you web development. You can get started with the <a href="http://freecodecamp.com/map" target="_blank">map</a>.
-<br><br>
-If you have any other questions, please post them in the discussion section. You can also join our WhatsApp group by filling <a href="https://goo.gl/sRbiXd" target="_blank"> our membership form</a>.
-       </p>
-      </div>
-
-      <div id="challenges">
+    <header>
+      <a class='banner' target="_blank" href="https://www.facebook.com/events/1798201603730103"><img src="https://fcc-hyd.github.io/codefights/img/banner.png"></a>
```

Comment:
@duttakapil I think so, I can make that change and what about rest of the images, should i download them and place in `img/challenges` folder and reference them??